L/Libxc/build_tarballs.jl

```diff
@@ -3,12 +3,12 @@
 using BinaryBuilder, Pkg
 
 name = "Libxc"
-version = v"5.0.0"
+version = v"5.1.0"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://gitlab.com/libxc/libxc/-/archive/5.0.0/libxc-5.0.0.tar.gz",
-                  "6b3be3cf6daf6b3eddf32d4077276eb9169531b42f98c2ca28ac85b9ea408493"),
+    ArchiveSource("https://gitlab.com/libxc/libxc/-/archive/5.1.0/libxc-5.1.0.tar.gz",
```

Comment:
```suggestion
    ArchiveSource("https://gitlab.com/libxc/libxc/-/archive/$(version)/libxc-$(version).tar.gz",
```

Reply:
Sure :smile: Makes it easier in the future, thanks @giordano.